jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -29,6 +30,19 @@ class JaxToTfTestCase(jtu.JaxTestCase):
 
   def setUp(self):
     super().setUp()
+
+    # Initialize categorizer
+    categorizer = jax2tf.jax2tf.Categorizer()
+    # Making sure the exit handler is not registered several times
+    atexit.unregister(categorizer.pprint_limitations)
+    # Register limitation summary atexit printing handler
+    atexit.register(categorizer.pprint_limitations, None)
```

Comment:
I suspect that this will register multiple printers, one for each test.

Reply:
I see that you are unregistering it first. Why not do everything at global level somehow: create the categorizer, register the printer? Maybe use this: https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

Reply:
Thanks, I was actually looking for something like that – but missed it